src/client/settings/settingsReducer.js

```diff
@@ -7,6 +7,8 @@ const initialState = {
   votePercent: 10000,
   showNSFWPosts: false,
   loading: false,
+  upvoteSetting: true,
+  rewardSetting: '50',
```

Comment:
I see this number '50' a lot throughout this PR, I feel like it should be a constant somewhere, like `export const INITIAL_REWARD_SETTING = '50'` then you can use that constant throughout where ever files need to be done